packages/algorithm/package.json

```diff
@@ -29,7 +29,6 @@
     "build:test": "tsc --build tests && cd tests && webpack",
     "clean": "rimraf ./lib && rimraf *.tsbuildinfo && rimraf ./types && rimraf ./dist",
     "clean:test": "rimraf tests/build",
-    "docs": "typedoc --options ../../typedoc.js src",
```

Comment:
I decided to get rid of the `docs` script in each individual package because it doesn't really make sense to me to build the docs for a single package in isolation of the others, especially since some of them depend on others in the monorepo.

Please let me know if my reasoning here is wrong.